src/Apps/IdentityVerification/IdentityVerificationPage.tsx

```diff
@@ -0,0 +1,101 @@
+import { Box, Button, Sans, Serif } from "@artsy/palette"
+import { IdentityVerificationPage_me } from "__generated__/IdentityVerificationPage_me.graphql"
+import { AppContainer } from "Apps/Components/AppContainer"
+import { SystemContextConsumer, useTracking } from "Artsy"
+import * as Schema from "Artsy/Analytics/Schema"
+import { ErrorPage } from "Components/ErrorPage"
+import React from "react"
+import { Meta, Title as HeadTitle } from "react-head"
+import { createFragmentContainer } from "react-relay"
+import { graphql } from "relay-runtime"
+
+interface Props {
+  me: IdentityVerificationPage_me
+}
+
+export const IdentityVerification: React.FC<Props> = ({ me }) => {
+  const { identityVerification } = me
+
+  // TODO: account for: expired, not present, completed, etc
+  if (!identityVerification || identityVerification.userID !== me.internalID) {
+    return <ErrorPage code={404} />
+  }
+
+  const { trackEvent } = useTracking()
+
+  const clickContinueToVerification = () => {
+    trackEvent({
+      context_page_owner_id: identityVerification.internalID,
+      action_type: Schema.ActionType.ClickedContinueToIdVerification,
+      context_page: Schema.PageName.IdentityVerificationPage,
+    })
+  }
+
+  return (
+    <SystemContextConsumer>
```

Comment:
Why is this needed?

Reply:
I figured it was something to include by convention.

Reply:
All apps mounted into the router have [this tree](https://github.com/artsy/reaction/blob/master/src/Artsy/Router/Boot.tsx) at the root, which means that if you need props from the system context can use the  [useSystemContext](https://github.com/artsy/reaction/blob/master/src/Artsy/SystemContext.tsx#L118-L124) hook.

Reply:
makes sense, will remove.